### PR TITLE
navbar improvement

### PR DESCRIPTION
**Pull request by SubhamB2003**
## Related Issue
issue solved: #166 

Closes: #[issue number]

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|      Original       |       Updated        |
| :-----------------: | :------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Review by SrijanShovit — COMMENTED** [review]
Hello SubhamB2003, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!<p></p>

File: yarn.lock
Comment (shreya024) [inline yarn.lock:1]: @SubhamB2003 Resolve the merge conflicts

Comment (SubhamB2003) [reply yarn.lock:1]: Done

**Comment by SubhamB2003** [comment]
@shreya024 Hey, can you please review it

**Review by shreya024 — CHANGES_REQUESTED** [review]
@SubhamB2003 Resolve the conflicts

**Comment by SubhamB2003** [comment]
@shreya024 done.

**Comment by shreya024** [comment]
> @shreya024 Can you please merge my pr?

@SubhamB2003 Take a pull,resolve merge conflicts and make sure your code adheres to the changed folder structure

**Comment by SubhamB2003** [comment]
@shreya024 Hey, the navbar issue is already fixed by someone 🥲

previously it showed this
image

but currently, it is showing this 
image

**Comment by SubhamB2003** [comment]
@shreya024 Please merge my pr

**Review by shreya024 — APPROVED** [review]

**Pull request merged**